src/OrbitGl/TimeGraph.cpp

```diff
@@ -481,6 +485,59 @@ void TimeGraph::ProcessMemoryTrackingTimer(const TimerInfo& timer_info) {
   }
 }
 
+void TimeGraph::ProcessCGroupAndProcessMemoryTrackingTimer(const TimerInfo& timer_info) {
+  int64_t cgroup_limit_bytes = orbit_api::Decode<int64_t>(timer_info.registers(
+      static_cast<size_t>(OrbitApp::CGroupAndProcessMemoryUsageEncodingIndex::kCGroupLimitBytes)));
+  int64_t cgroup_rss_bytes = orbit_api::Decode<int64_t>(timer_info.registers(
+      static_cast<size_t>(OrbitApp::CGroupAndProcessMemoryUsageEncodingIndex::kCGroupRssBytes)));
+  int64_t cgroup_mapped_file_bytes =
+      orbit_api::Decode<int64_t>(timer_info.registers(static_cast<size_t>(
+          OrbitApp::CGroupAndProcessMemoryUsageEncodingIndex::kCGroupMappedFileBytes)));
+  int64_t process_rss_anon_kb = orbit_api::Decode<int64_t>(timer_info.registers(
+      static_cast<size_t>(OrbitApp::CGroupAndProcessMemoryUsageEncodingIndex::kProcessRssAnonKb)));
+
+  if (cgroup_limit_bytes == kMissingInfo || cgroup_rss_bytes == kMissingInfo ||
+      cgroup_mapped_file_bytes == kMissingInfo || process_rss_anon_kb == kMissingInfo) {
+    return;
+  }
+
+  constexpr double kMegabytesToBytes = 1024.0 * 1024.0;
+  constexpr double kMegabytesToKilobytes = 1024.0;
+  orbit_gl::CGroupAndProcessMemoryTrack* track = track_manager_->GetCGroupAndProcessMemoryTrack();
+  if (track == nullptr) {
+    const std::array<std::string, orbit_gl::kCGroupAndProcessMemoryTrackDimension> kSeriesNames = {
+        absl::StrFormat("Process [%s] Resident Anonymous Memory", capture_data_->process_name()),
+        "Other Processes Resident Anonymous Memory",
+        absl::StrFormat("CGroup [%s] Mapped File", timer_info.cgroup_name()),
+        absl::StrFormat("CGroup [%s] Unused", timer_info.cgroup_name())};
```

Comment:
From the screenshot, these seem to be very wide, potentially going past the width of the track even when the window is quite large.